bookwyrm/models/list.py

```diff
@@ -43,6 +45,7 @@ class List(OrderedCollectionMixin, BookWyrmModel):
         through="ListItem",
         through_fields=("book_list", "book"),
     )
+    embed_key = models.UUIDField(unique=True, null=True, editable=False)
```

Comment:
I think you can set `default=uuid4` here, and you won't have to compute it manually or add it to the migration

Reply:
From what I tested, the `default` value is not unique. I may have overlooked something, but the addition in the migration was the best way I saw to handle the uniqueness requirement.

Reply:
Weird! I would never have suspected it wouldn't be unique -- thanks for looking into it